packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx

```diff
@@ -95,7 +95,7 @@ export const MultiValuesInput: React.FunctionComponent<MultiValuesInputProps> =
                         innerInputClasses={innerInputClasses}
                         disabledTooltip={isTooltipRequired && disabledTooltipTitle ? disabledTooltipTitle : ''}
                         labelTitle={index === 0 ? inputProps?.labelTitle : ''}
-                        validate={index === 0 ? inputProps?.validate : (value: string) => true}
+                        validate={inputProps?.validate}
```

Comment:
Is there a change this is breaking a form somewhere in the admin? :thinking: 
(My guess is not but i'm not that familiar with the multiValueInput ...)

Reply:
Honestly, I'm not very familiar with this component either. As far as I'm concerned, this will force all the input values to pass the validation instead of only the first one which makes sense. Let's summon a senpai then @gdostie

Reply:
If you are in doubt you can link this branch with the admin and head to the internal events panel of the settings. There is a multiline input with IP validation there and you will be able to make sure the change doesn't change anything.